doc/releases/changelog-0.38.0.md

```diff
@@ -4,112 +4,396 @@
+  if they are present in the Qiskit noise model.
+
+  Make sure to `pip install pennylane-qiskit` to access this new feature!
+
+<h4>Substantial upgrades to mid-circuit measurements using tree-traversal 🌳</h4>
+
+* The `"tree-traversal"` algorithm for mid-circuit measurements (MCMs) on `default.qubit` has been internally redesigned for better 
+  performance.
+  [(#5868)](https://github.com/PennyLaneAI/pennylane/pull/5868)
+
+  In the last release (v0.37), we introduced the tree-traversal MCM method, which was implemented in 
+  a recursive way for simplicity. However, this had the unintended consequence of very deep [stack calls](https://en.wikipedia.org/wiki/Call_stack) 
+  for circuits with many MCMs, resulting in [stack overflows](https://en.wikipedia.org/wiki/Stack_overflow) 
+  in some cases. With this release, we've refactored the implementation of the tree-traversal method 
+  into an iterative approach, which solves those inefficiencies when many MCMs are present in a circuit.
+ 
+* The `tree-traversal` algorithm is now compatible with analytic-mode execution (`shots=None`).
+  [(#5868)](https://github.com/PennyLaneAI/pennylane/pull/5868)
+
+  ```python
+  dev = qml.device("default.qubit")
+
+  n_qubits = 5
+
+  @qml.qnode(dev, mcm_method="tree-traversal")
+  def circuit():
+      for w in range(n_qubits):
+          qml.Hadamard(w)
+      
+      for w in range(n_qubits - 1):
+          qml.CNOT(wires=[w, w+1])
+
+      for w in range(n_qubits):
+          m = qml.measure(w)
+          qml.cond(m == 1, qml.RX)(0.1967 * (w + 1), w)
+
+      return [qml.expval(qml.Z(w)) for w in range(n_qubits)]
+  ```
+
+  ```pycon
+  >>> circuit()
+  [tensor(0.00964158, requires_grad=True),
+   tensor(0.03819446, requires_grad=True),
+   tensor(0.08455748, requires_grad=True),
+   tensor(0.14694258, requires_grad=True),
+   tensor(0.2229438, requires_grad=True)]
+  ```
 
 <h3>Improvements 🛠</h3>
 
-* Counts measurements with `all_outcomes=True` can now be used with jax jitting. Measurements
-  broadcasted across all available wires (`qml.probs()`) can now be used with jit and devices that
-  allow variable numbers of wires (`qml.device('default.qubit')`).
-  [(#6108)](https://github.com/PennyLaneAI/pennylane/pull/6108/)
+<h4>Creating spin Hamiltonians</h4>
+
+* Three new functions are now available for creating commonly-used spin Hamiltonians in PennyLane:
+  [(#6106)](https://github.com/PennyLaneAI/pennylane/pull/6106)
+  [(#6128)](https://github.com/PennyLaneAI/pennylane/pull/6128)
+
+  * `qml.spin.transverse_ising`: creates the [transverse-field Ising model](https://en.wikipedia.org/wiki/Transverse-field_Ising_model) Hamiltonian.
+  * `qml.spin.heisenberg`: creates the [Heisenberg model](https://en.wikipedia.org/wiki/Quantum_Heisenberg_model) Hamiltonian.
+  * `qml.spin.fermi_hubbard`: creates the [Fermi-Hubbard model](https://en.wikipedia.org/wiki/Hubbard_model) Hamiltonian.
```

Comment:
This section is not rendering correctly, and I can't tell why 🤔 

![image](https://github.com/user-attachments/assets/0facd730-dd8c-4319-a8fb-29970ce4856e)

Reply:
![Screenshot 2024-08-30 at 5 49 59 PM](https://github.com/user-attachments/assets/a1555e06-610c-455d-88ef-886a01eadc52)
resolved! 🤷